README.md

```diff
@@ -0,0 +1,110 @@
+# Chivalry 2 Unchained Launcher (C2UL)
+
+- [What is the Chivalry 2 Unchained Launcher (C2UL)?](#what-is-the-chivalry-2-unchained-launcher-c2ul)
+- [Features](#features)
+- [Join the Community](#community)
+- [Installation](#installation)
+  - [Installation Instructions](#installation-instructions)
+- [General Use Instructions](#general-use-instructions)
+  - [How to Use the Tool](#how-to-use-the-tool)
+  - [Troubleshooting](#troubleshooting)
+  - [Note](#note)
+- [Developer-Focused App Structure and Details](#developer-focused-app-structure-and-details)
+- [Assets](#assets)
+- [License](#license)
+- [Contributing](#contributing)
+
+## What is the Chivalry 2 Unchained Launcher (C2UL)?
+
+The Chivalry 2 Unchained Launcher (C2UL) is a graphical user interface (GUI) tool designed to enhance the Chivalry 2 gaming experience. By providing an easy way to launch the game, manage mods, and customize settings, C2UL expands the capabilities of Chivalry 2, fostering community growth and enriching the overall player experience.
+
+## Features
+
+- **Launch Chivalry 2**: Easily launch Chivalry 2 with custom configurations.
+- **Mod Management**: Download and manage mods for Chivalry 2.
+- **Customization**: Customize the launcher with various settings and preferences.
+
+## Community
+
+Join the [Chivalry 2 Unchained community on Discord](https://discord.gg/chiv2unchained) to connect with other players, get support, and participate in discussions.
+
+## Installation
+
+1. [Download the latest binary from the releases page](https://github.com/Chiv2-Community/C2GUILauncher/releases).
+2. Open the downloaded file to install the C2UL.
+
+### Installation Instructions
+
+- Rename `Chivalry2Launcher.exe` to `Chivalry2Launcher-ORIGINAL.exe` -- This is important. The name is significant.
+- Put the downloaded `Chivalry2Launcher.exe` in place of the old one.
+- Launch the game normally (either from your steam library or from Epic Games Store).
+
+## General Use Instructions
+
+### How to Use the Tool
+
+1. **Open the Launcher**: Run the C2UL from the installed location.
+2. **Select Installation Type**: If the installation type is not auto-detected, select it manually from the provided options.
```

Comment:
I think we can omit this. If it fails to autodetect you'll get a warning on launch and it'll only fail autodetect for people with a custom location. I think people who are tech savvy enough to put chiv2 in a custom install location will be able to work this out from the dialogue built in to the launcher.

Reply:
I don't think including it to set people up with the expectation is a bad thing personally

Reply:
Fair enough. Should probably include that this setting is on the settings tab tho